assets/src/apps/admin.tsx

```diff
@@ -35,6 +36,7 @@ const routes: [string, string, ComponentType][][] = [
     ["gl-eink-v2-screens", "GL E-ink", GLEinkV2ScreensTable],
     ["pre-fare-v2-screens", "Pre-Fare", PreFareV2ScreensTable],
     ["busway-v2-screens", "Sectional", BuswayV2ScreensTable],
+    ["elevator-v2-screens", "Elevator", ElevatorV2ScreensTable],
```

Comment:
Er, to clarify (re: you also moved "Sectional"), I'm suggesting we keep them alphabetized by label, not by app ID, since that's what users of the admin UI are looking at to navigate.

Reply:
Ah woops my bad